drivers/overlay/ovmanager/ovmanager.go

```diff
@@ -57,7 +57,7 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		config:   config,
 	}
 
-	d.vxlanIdm, err = idm.New(nil, "vxlan-id", vxlanIDStart, vxlanIDEnd)
```

Comment:
@aboch the value of vxlanIDStart must be 4096. But I think this fix must be enhanced to use `vxlanIDStart` when the user let ovmanager to choose a random id. I will push an update.